config/locales/diaspora/en.yml

```diff
@@ -683,6 +683,7 @@ en:
         subject: "%{name} has mentioned you on diaspora*"
         limited_post: "You were mentioned in a limited post."
     private_message:
+        subject: "Diaspora: Private Message"
```

Comment:
The subject for `also_commented` notifications for private posts is
> There's a new comment on a post you commented

so maybe 
> There’s a new private message for you